app/ExposureHistoryContext.tsx

```diff
@@ -61,24 +66,34 @@ const ExposureHistoryProvider = ({
     exposureInfoSubscription,
     toExposureHistory,
   } = exposureEventsStrategy;
+
+  const featureFlags = useSelector(
+    (state: RootState) => state.featureFlags?.flags || {},
+  );
+  const exposureMode = !!featureFlags[FeatureFlagOption.EXPOSURE_MODE];
```

Comment:
Can rename this to the functionality the feature flag does (maybe rename the feature flag) 
```suggestion
  const mockExposureNotification = !!featureFlags[FeatureFlagOption.MOCK_EXPOSURE];
```